frontend/src/components/CreateCsv.jsx

```diff
@@ -0,0 +1,50 @@
+import { Button } from "antd";
+
+const CreateCsv = ({ tableData, selectedSalaryDate, finnishDateNoDots }) => {
+  const generateCsv = () => {
+    const titleKeys = Object.keys(tableData[0]);
+    const transformedData = [];
+    transformedData.push(titleKeys);
+
+    tableData.forEach((row) => {
+      let transformedRow = [];
+
+      Object.values(row).forEach((value) => {
+        let transformedValue = value;
+
+        if (typeof value === "number" && value.toString().includes(".")) {
+          transformedValue = value.toString().replace(".", ",");
+        }
+        transformedRow.push(transformedValue);
+      });
+
+      transformedData.push(transformedRow);
+    });
+    let csvContent = "\ufeff";
```

Comment:
It would be nice to add a comment why there is this in the beginning of the csvContent. It's not easy to figure out just by reading the code.